ci/cases/gfsv17/marinehyb.yaml

```diff
@@ -7,6 +7,7 @@ base:
   DO_JEDIATMENS: "NO"
   DO_JEDIOCNVAR: "YES"
   DO_JEDISNOWDA: "NO"
+  DO_STARTMEM_FROM_JEDIICE: "YES"
```

Comment:
This is actually a problem for the very first cycle. We'll need some kind of logic if the analysis isn't there (or maybe we need to prep initial conditions so that there is a copy of background in the relevant directory? @guillaumevernieres.

Reply:
Yes, we'll have to update how we prepare the IC's.